troubleshooting/incrementally_configuring_images.md

```diff
@@ -0,0 +1,297 @@
+# Incrementally configuring images
+
+When IUF or SAT `bootprep` are failing to configure an image, this procedure can be used to incrementally configure an image.
+This avoids re-running the build and configuration steps that have succeeded, which would occur if IUF or SAT `bootprep` were re-run.
+
+The following example assumes the user is creating a `Compute` image, but can be adapted for other image types.
+
+## Setting up
+
+1. (`ncn-mw#`) Find the path to the IUF logs printed by the `iuf` command. It will look like this:
+
+   ```text
+   2023-11-01T14:06:36.128255Z INFO All logs will be stored in /etc/cray/upgrade/csm/iuf/cos-products-20231031/log/20231101140636
+   ```
+
+   See `install.log` in this directory when subsequent steps reference the IUF logs.
+
+1. (`ncn-mw#`) Find the SAT logs by looking for a line like the following in the IUF logs:
```

Comment:
This step looks nearly identical to one a few steps later. I gather that you are trying to identify two different lines, but both instructions describe the target as "the SAT logs". I think you should add some verbiage to differentiate the two lines. Something like "Find the SAT logs in the `prepare-managed-images` section ... " and "Find the SAT logs in the `update-managed-cfs-config` section ..."